test/ui-test/src/screens/SettingsScreen.py

```diff
@@ -417,9 +417,9 @@ def check_backup_seed_phrase_workflow(self):
         started_at = time.monotonic()
         while wait_for_is_visible(BackupSeedPhrasePopup.REVEAL_SEED_PHRASE_BUTTON.value, verify=False):
             try:
-                hover(BackupSeedPhrasePopup.REVEAL_SEED_PHRASE_BUTTON.value)
+                # hover(BackupSeedPhrasePopup.REVEAL_SEED_PHRASE_BUTTON.value)
                 click_obj_by_name(BackupSeedPhrasePopup.REVEAL_SEED_PHRASE_BUTTON.value)
-            except (LookupError, RuntimeError):
+            except (LookupError, RuntimeError, AssertionError) as error:           
```

Comment:
If we remove the hover,  the AssertionError could not be thrown, so we don't need to catch it

Reply:
correct, sorry , i overlooked, fixed now